app/src/main/kotlin/com/intive/picover/parties/di/PartiesModule.kt

```diff
@@ -17,4 +20,9 @@ abstract class PartiesModule {
 	@IntoMap
 	@ScreenModelFactoryKey(PartyDetailsViewModel.Factory::class)
 	abstract fun bindPartyDetailsViewModelFactory(factory: PartyDetailsViewModel.Factory): ScreenModelFactory
+
+	@Binds
+	@IntoMap
+	@ScreenModelKey(PartiesViewModel::class)
+	abstract fun bindPartiesViewModel(viewModel: PartiesViewModel): ScreenModel
```

Comment:
Turns out that for regular screen models binding is also needed to be able to inject it with hilt :/

Reply:
As you already mentioned, it's a good candidate for KSP

Reply:
We'll see how it works with Koin, maybe the integration will be less painful